docs/siem/timeline/timeline-ui-overview.asciidoc

```diff
@@ -0,0 +1,126 @@
+[[timelines-ui]]
+[role="xpack"]
+= Investigating events in Timeline
+
+Use Timeline as your workspace for investigations and threat hunting.
+Data from multiple indices can be added to a Timeline, which enables
+investigating complex threats.
+
+You can drag or send items of interest to Timeline to create the query you need
+to investigate an alert or event. You can add items from tables and histograms
+on the *Overview*, *Detections*, *Hosts* and *Network* pages, as well as from
+within a Timeline itself. Additionally, you can add a query directly in Timeline
+(click `+ Add field`).
+
+[role="screenshot"]
+image::images/timeline-ui.png[]
+
+Timelines are responsive, and they persist as you move through the {es-sec-app}
+collecting data. Auto-saving ensures that the results of your investigation are
+available for review by other analysts and incident response teams. To record
+and share your findings with others, attach your Timeline to a
```

Comment:
could we emphasis on `attach timeline  templates`, here instead of just timeline.
We do allow users link default timeline to rule, but in 7.9 we only show timeline templates as options when creating rules.